### ISSUE TITLE
Global settings

### ISSUE THREAD

**Issue by battle-panda** [pandacorps/twist-design#10]
We need to figure out global settings, including:
- Configuring theme (light vs dark)
- more to come...

**Comment by compiledpanda** [comment]
Agreed. For now we should use the system defaults for light and dark theming. Here is an article that has some options on how to do it: https://medium.com/@pmutisya/dark-mode-in-flutter-3742062f9f59
